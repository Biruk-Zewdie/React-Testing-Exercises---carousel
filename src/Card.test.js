import React from "react";
import {render} from "@testing-library/react"
import Card from "./Card";


//smoke test
test ("renders without crushing", () => {
    render (<Card caption= "testing image"  src="testing.com" currNum={1} totalNum={3}/>)
})

//snapshot test 
test ("should match to snapshot ", () =>{
    const {asFragment, debug} = render (<Card caption= "testing image"  src="testing.com" currNum={1} totalNum={3}/>)
    debug();
    expect (asFragment()).toMatchSnapshot();
})