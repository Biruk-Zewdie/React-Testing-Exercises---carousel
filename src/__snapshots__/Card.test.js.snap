// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match to snapshot  1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image
    </h4>
    <img
      alt="testing image"
      class="Card-image"
      src="testing.com"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
